src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml

```diff
@@ -34,375 +32,326 @@
-                            <label asp-for="BirthMonth" class="govuk-label govuk-date-input__label">Month</label>
-                            <input asp-for="BirthMonth" min="1" max="12" class="govuk-input govuk-date-input__input govuk-input--width-2" type="number">
-                        </div>
-                    </div>
-                    <div class="govuk-date-input__item">
-                        <div class="govuk-form-group">
-                            <label asp-for="BirthYear" class="govuk-label govuk-date-input__label">Year</label>
-                            <input asp-for="BirthYear" min="1900" max="9999" class="govuk-input govuk-date-input__input govuk-input--width-4" type="number">
-                        </div>
-                    </div>
-                </div>
-            </fieldset>
-        </div>
-
-        @if (AuthorizationService.IsAuthorized(ProviderFeature.FlexiblePaymentsPilot))
-        {
-            <h2 class="govuk-heading-m">Pilot Program</h2>
-
-            <div class="govuk-form-group @Html.AddClassIfPropertyInError(x => x.IsOnFlexiPaymentPilot, "govuk-form-group--error")">
-                <fieldset class="govuk-fieldset">
-                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
-                        Will this apprentice be part of the Flexible Payments pilot program?
-                    </legend>
-                    <div class="govuk-radios">
-                        @Html.ValidationMessageFor(m => m.IsOnFlexiPaymentPilot, null, new {@class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.IsOnFlexiPaymentPilot)})
-                        <div class="govuk-radios__item">
-                            <input class="govuk-radios__input" id="IsOnFlexiPaymentPilot" name="IsOnFlexiPaymentPilot" type="radio" value="true" @(Model.IsOnFlexiPaymentPilot.HasValue && Model.IsOnFlexiPaymentPilot.Value ? "checked" : "")>
-                            <label class="govuk-label govuk-radios__label" for="IsOnFlexiPaymentPilot">
-                                Yes
-                            </label>
-                        </div>
-                        <div class="govuk-radios__item">
-                            <input class="govuk-radios__input" id="IsOnFlexiPaymentPilot-no" name="IsOnFlexiPaymentPilot" type="radio" value="false" @(Model.IsOnFlexiPaymentPilot.HasValue && !Model.IsOnFlexiPaymentPilot.Value ? "checked" : "")>
-                            <label class="govuk-label govuk-radios__label" for="IsOnFlexiPaymentPilot-no">
-                                No
-                            </label>
-                        </div>
-                    </div>
-                </fieldset>
-            </div>
-        }
-        else
-        {
-            <input type="hidden" asp-for="IsOnFlexiPaymentPilot" value="false" />
-        }
-    }
+@if (Model is EditDraftApprenticeshipViewModel)
+{
+    <h1 class="govuk-heading-xl">Edit personal details</h1>
```

Comment:
In the prototype/mockups I've seen, I think this header is just "Apprentice personal details" (no "add" or "edit")